sklearn/datasets/samples_generator.py

```diff
@@ -791,7 +791,7 @@ def make_blobs(n_samples=100, n_features=2, centers=3, cluster_std=1.0,
 
 
 def make_friedman1(n_samples=100, n_features=10, noise=0.0, random_state=None):
-    """Generate the "Friedman \#1" regression problem
+    """Generate the "Friedman \\#1" regression problem
```

Comment:
I find these escapes strange. I wonder if a git blame would explain why they were inserted and whether they are still needed

Reply:
git blame says it was this commit https://github.com/scikit-learn/scikit-learn/commit/b5950ad070fe1d5a44d52ecb4275dac970ec91ca.

Reply:
The result does not change when the escape is removed. The docstring of http://scikit-learn.org/stable/modules/generated/sklearn.feature_selection.RFECV.html also writes `Friedman #1` without an escape and this does not seem to cause problems.

Reply:
It might be a historical thing. I'd happily see them go.

Reply:
The escapes are gone now.